telemetry/opting-out-of-telemetry.adoc

```diff
@@ -0,0 +1,21 @@
+[id="opting-out-of-telemetry"]
+= Opting out of Telemetry
+include::modules/common-attributes.adoc[]
+:context: telemetry
+
+toc::[]
+
+You might need to opt out of Telemetry for your cluster.
+
+[NOTE]
+====
+On OpenShift Dedicated, Telemetry is always enabled. You cannot opt out.
+====
+
+.Prerequisites
+
+* An installed `oc` CLI tool.
```

Comment:
We're changing the way that we use this in prereqs. Jason has an open PR with the details.